docs/introduction/start-building-your-application.md

```diff
@@ -158,7 +158,7 @@ After all, your production server is the one that matters the most.
 First, run `php -v` on your server to find our the exact version, for example:
 
 ```no-highlight
-PHP 8.1.3 (cli) (built: Mar  1 2022 09:21:02) ( NTS )
+PHP 8.3.2 (cli) (built: Mar  1 2022 09:21:02) ( NTS )
```

Comment:
The built part should be updated (or in my opinion better deleted)

Reply:
removed